sui/its.js

```diff
@@ -5,13 +5,18 @@ const { addBaseOptions, addOptionsToCommands, getWallet, printWalletInfo, broadc
 const { bcs } = require('@mysten/sui/bcs');
 
 const SPECIAL_CHAINS_TAGS = {
-    ALL_EVM: 'all-evm', // All EVM chains that have InterchainTokenService deployed
+    ALL: 'all',
 };
 
+function getAllTrustedChains(config) {
```

Comment:
This should be a common/util that gets used in EVM `deploy-its.js` as well
https://github.com/axelarnetwork/axelar-contract-deployments/blob/feat/ITS-hub-upgrade-v1.1.0/evm/deploy-its.js#L135